### PR TITLE
HSDO-425 #added speaker field, speaker link, and speaker mapping to importer

### PR DESCRIPTION
**Pull request by pookmish**
# To Test
1. Checkout branch speaker-field
2. Clear cache & revert feature.
3. Manually create events or import events that have a speaker (.../People/person in the xml).

### CODE REVIEW

**Comment by pookmish** [comment]
Just cleaned up merge conflicts

File: stanford_events_importer.info [hunk 1]
```diff
@@ -8,10 +8,11 @@ dependencies[] = ctools
 dependencies[] = date
 dependencies[] = date_all_day
 dependencies[] = date_popup
-dependencies[] = features (>=2.0)
```
Comment (sherakama) [inline stanford_events_importer.info:11]: Features strips these specific version statements. Please restore them.

File: stanford_events_importer.strongarm.inc [hunk 2]
```diff
@@ -193,6 +193,12 @@ function stanford_events_importer_strongarm() {
       'stanford_8_col_banner' => array(
         'custom_settings' => FALSE,
       ),
+      'stanford_cap' => array(
```
Comment (sherakama) [inline stanford_events_importer.strongarm.inc]: We can probably pull these out. They are not needed.

**Comment by sherakama** [comment]
Code looks good. Please add some CSS to format the output of the speakers.

**Pull request merged**